rails/locale/az.yml

```diff
@@ -116,13 +120,16 @@ az:
       invalid: yanlışdır
       less_than: "%{count}-dən kiçik olmalıdır"
       less_than_or_equal_to: kiçik və ya %{count}-ə bərabər olmalıdır
+      model_invalid: "Yoxlama uğursuz oldu: %{errors}"
       not_a_number: rəqəm deyil
       not_an_integer: tam rəqəm olmalıdır
       odd: tək olmalıdır
+      required: mövcud olmalıdır
       taken: artıq mövcuddur
       too_long: çox uzundur (%{count} simvoldan çox olmalı deyil)
       too_short: çox qısadır (%{count} simvoldan az olmalı deyil)
       wrong_length: uzunluqu səhvdir (%{count} simvol olmalıdır)
+      other_than: "%{count}-dən fərqli olmalıdır"
```

Comment:
@NARKOZ as you already know, the ablative case is expressed with two suffixes (_dan_  and _dən_) according to the last syllabus of the word. For numbers, like 6 (_az:_ altı), 9 (_az_: doqquz) etc. the correct ablative case should be formed by _-dan_.

Reply:
Can you provide alternative translation without the suffix?

Reply:
Yes, please [review this](https://github.com/svenfuchs/rails-i18n/pull/789). 

P.S. I were not able to generate a meaningful message for `other_than`